src/__internal__/popover/popover.component.tsx

```diff
@@ -45,6 +45,8 @@ export interface PopoverProps {
   disablePortal?: boolean;
   // Reference element, children will be positioned in relation to this element - should be a ref shaped object
   reference: React.RefObject<HTMLElement>;
+  // Determines if the popover is currently open/visible or not. Defaults to true.
+  isOpen?: boolean;
 }
 // TODO: Remove TempModalContext after modal has been converted to TS
 type TempModalContext = {
```

Comment:
While you're updating this file can you update this to use the `ModalContext` as the refactor of `Modal` is now in master?